sycl/include/CL/sycl/usm/usm_allocator.hpp

```diff
@@ -107,35 +122,26 @@ class usm_allocator {
         "Device pointers do not support address on host");
   }
 
-  // Allocate memory
-  template <
-      usm::alloc AllocT = AllocKind,
-      typename std::enable_if<AllocT == usm::alloc::host, int>::type = 0>
+  /// Allocates memory.
+  ///
+  /// @param Size is a count of elements to allocate memory for.
```

Comment:
Wondering if `Size` is the right name for the parameter, which is `a count of elements ...`.
`NumOfElements`?